applications/livetap/Chart.yaml

```diff
@@ -1,8 +1,13 @@
 apiVersion: v2
-name: cadc-tap
+name: live-tap
```

Comment:
This should probably be `livetap`, I think, to match the application name. I think elsewhere we've made the chart names match the application names.